geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/webpack.apps.js.mako_tmpl

```diff
@@ -72,5 +72,5 @@ module.exports = {
     alias: {
       {{package}}: path.resolve(__dirname, '{{package}}_geoportal/static-ngeo/js'),
     }
-  }
+  },
```

Comment:
Why?

Reply:
To don't need to modify this line if we add an other attribute :-)